python/tvm/relay/op/_transform.py

```diff
@@ -304,6 +305,31 @@ def compute_argwhere(attrs, inputs, output_type, _):
     new_output_type = tvm.relay.ty.TensorType(output_shape, "int32")
     return [topi.argwhere(new_output_type, inputs[0])]
 
+@_reg.register_compute("strided_set")
+def compute_strided_set(attrs, inputs, output_type, _):
+    """Compute definition of strided_set"""
+    begin = attrs.begin
+    end = attrs.end
+    strides = attrs.strides
+    n = len(inputs[0].shape)
+    ls = len(strides)
+    if ls < n:
+        strides = list(strides)
+        strides[ls:] = [1] * (n - ls)
+    lb = len(begin)
+    if lb < n:
+        begin = list(begin)
+        for i in range(lb, n):
+            begin.append(0 if strides[i] >= 0 else inputs.shape[i])
```

Comment:
same here

Reply:
This is a bit different because the value to append might be different at each index.  It depends on the stride for that index.  I'm not sure how to shorten that.

Reply:
It’s fine, no worries :-)